dali/pipeline/data/buffer.cc

```diff
@@ -14,11 +14,27 @@
 
 #include "dali/pipeline/data/buffer.h"
 #include "dali/pipeline/data/backend.h"
+#include "dali/core/mm/memory.h"
 
 namespace dali {
 
 // this is to make debug builds happy about kMaxGrowthFactor
 template class Buffer<CPUBackend>;
 template class Buffer<GPUBackend>;
 
+
+DLL_PUBLIC shared_ptr<uint8_t> AllocBuffer(size_t bytes, bool pinned, GPUBackend *) {
+  (void)pinned;  // this is less-than-ideal design
```

Comment:
nitpick: we can maybe omit the name in the signature, and avoid this cast

Reply:
Done.